modelling/src/main/java/org/axonframework/modelling/saga/metamodel/SagaMetaModelFactory.java

```diff
@@ -26,8 +32,33 @@ public interface SagaMetaModelFactory {
      * characteristics of the given type.
      *
      * @param sagaType The saga class to be inspected
-     * @param <T> The saga type
+     * @param <T>      The saga type
      * @return Model describing the capabilities and characteristics of the inspected saga class
      */
     <T> SagaModel<T> modelOf(Class<T> sagaType);
+
+    /**
+     * Returns an Interceptor Chain of annotated interceptor methods for the given {@code sagaType}. The given chain
+     * will invoke all relevant interceptors in an order defined by the handler definition.
+     *
+     * @param sagaType The saga class to be inspected
+     * @param <T>      The saga type
+     * @return an interceptor chain that invokes the interceptor handlers
+     */
+    default <T> MessageHandlerInterceptorMemberChain<T> chainedInterceptor(Class<T> sagaType) {
+        return NoMoreInterceptors.instance();
+    }
+
+    class NoMoreInterceptors<T> implements MessageHandlerInterceptorMemberChain<T> {
```

Comment:
I think it's fine to extract the `private static` `NoMoreInterceptors` class from the `AnnotatedHandlerInspector`, so that it can be reused. 
Making this adjustment results in that you can remove this interface and the test class' duplication, too, of course.

What do you think?

Reply:
Done